<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" ignored="0" total="1" passed="0">
  <reporter-output>
  </reporter-output>
  <suite name="Default suite" duration-ms="18288" started-at="2019-01-25T06:16:02Z" finished-at="2019-01-25T06:16:20Z">
    <groups>
    </groups>
    <test name="Default test" duration-ms="18288" started-at="2019-01-25T06:16:02Z" finished-at="2019-01-25T06:16:20Z">
      <class name="com.Testcases.ELTC_012">
        <test-method status="PASS" signature="setUpBeforeClass()[pri:0, instance:com.Testcases.ELTC_012@3bfdc050]" name="setUpBeforeClass" is-config="true" duration-ms="20" started-at="2019-01-25T06:16:02Z" finished-at="2019-01-25T06:16:02Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUpBeforeClass -->
        <test-method status="PASS" signature="setUp()[pri:0, instance:com.Testcases.ELTC_012@3bfdc050]" name="setUp" is-config="true" duration-ms="11626" started-at="2019-01-25T06:16:02Z" finished-at="2019-01-25T06:16:14Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method status="FAIL" signature="pswdrecoverTest()[pri:0, instance:com.Testcases.ELTC_012@3bfdc050]" name="pswdrecoverTest" duration-ms="4884" started-at="2019-01-25T06:16:14Z" finished-at="2019-01-25T06:16:19Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [Password sent to your mail id] but found [This platform was unable to send the email. Please contact Site Admin for more information.]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [Password sent to your mail id] but found [This platform was unable to send the email. Please contact Site Admin for more information.]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertEqualsImpl(Assert.java:137)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:453)
at org.testng.Assert.assertEquals(Assert.java:463)
at com.Testcases.ELTC_012.pswdrecoverTest(ELTC_012.java:63)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:580)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:716)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:988)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- pswdrecoverTest -->
        <test-method status="PASS" signature="tearDown()[pri:0, instance:com.Testcases.ELTC_012@3bfdc050]" name="tearDown" is-config="true" duration-ms="1745" started-at="2019-01-25T06:16:19Z" finished-at="2019-01-25T06:16:20Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- com.Testcases.ELTC_012 -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
